pages/index.js

```diff
@@ -45,20 +45,26 @@ export default () => {
     };
 
     const compare = () => {
-        localStorage.setItem('ignoreParams', ignore);
-        const ignoreParams = ignore.replace(' ', '').split(',');
-        const [diff, eq] = compareUrls(firstUrl, secondUrl, ignoreParams);
+        let ignoreParams = '';
+
+        try {
```

Comment:
Because we can't use `localStorage.setItem` in private tab Safari.
https://stackoverflow.com/questions/14555347/html5-localstorage-error-with-safari-quota-exceeded-err-dom-exception-22-an